### PR TITLE
Create Adelaide.md

### PR DESCRIPTION
**Pull request by jq326**
First draft of Adelaide file

### Description of Change

- First creation of Adelaide file :smiley:

### Test results

- No testing yet

### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

File: Adelaide.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Adelaide
+
+## Foodie tips
+
+## Wine news
```
Comment (jq326) [inline Adelaide.md:5]: We need more detail here

File: Adelaide.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Adelaide
+
+## Foodie tips
+
```
Comment (jq326) [inline Adelaide.md:4]: My review comment

Comment (jq326) [reply Adelaide.md:4]: second comment

**Review by jq326 — COMMENTED** [review]
looking foward to more info

File: Adelaide.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Adelaide
```
Comment (mj580) [inline Adelaide.md:1]: great stuff!

**Review by mj580 — APPROVED** [review]
very nice, indeed

**Review by pm700 — APPROVED** [review]
looks good

**Comment by BernardVillar** [comment]
need pictures

**Pull request merged**